docs/authentication/overview.mdx

```diff
@@ -1,5 +1,64 @@
 ---
-sanity_slug: /docs/authentication/overview
+title: Sign-up & Sign-in
+description: Learn how to configure authentication and user management for your Clerk application.
 ---
 
-# This is a stub
+# Sign-up & Sign-in
+
+Clerk supports multiple authentication strategies so that you can implement the strategy that makes sense for *your* users. You can use Clerk's [hosted pages](/docs/authentication/using-clerk-hosted-pages), [pre-built components](/docs/authentication/components/overview), or [build your own custom flows](/docs/custom-flows/overview).
+
+## Configuration
+
+Clerk's configuration settings affect how the users of your application can [sign up](/docs/authentication/configuration/sign-up-options) and [sign in](/docs/authentication/configuration/sign-in-options) and which attributes are editable via their user profile. You can also manage [user sessions](/docs/authentication/configuration/session-options), [control who gets access](/docs/authentication/configuration/allowlist) to your application, and [customize the email & SMS messages](/docs/authentication/configuration/email-options) that are sent by Clerk during authentication flows. All of these settings can be found under the **User & Authentication** section in your [Clerk Dashboard](https://dashboard.clerk.dev/).
```

Comment:
This section is titled ##Configuration so I added references to the Configuration sections on the left, so that this overview is clear on what information the /docs/authentication/configuration section is going to be providing.